lib/Modules/HyperVCIHooks/HyperVCIHooks.psm1

```diff
@@ -1345,7 +1354,12 @@ function Start-InstallHook {
     $zuulBranch = Get-JujuCharmConfig -scope 'zuul-branch'
 
     Write-JujuLog "Initializing the environment"
-    Initialize-Environment -BranchName $zuulBranch -BuildFor $zuulProject
+    if (!$zuulProject) {
+        Initialize-Environment -BranchName "master" -BuildFor "openstack/none"
```

Comment:
You have defalt params / can check if param is defined. I see no reason to pass a "text" param here unless you use it in other places afterwards and it's required (though does not seem so to me).